### ISSUE TITLE
Houston could not recognize localized screenshots

### ISSUE THREAD

**Issue by ryonakano** [elementary/houston#727]
Recently I found developers can provide localized screenshots to AppCenter in https://github.com/elementary/appcenter/issues/933 and I [added localized screenshots](https://github.com/ryonakano/reco/commit/1d2457d4aa4ad8890d7d2722a634a387640aa229) to my app.

However, [the build fails](https://travis-ci.org/ryonakano/reco/builds/504548527?utm_source=github_status&utm_medium=notification) after that commit. Here is the log which Houston output:

```
# Missing image tag in screenshot
Log: Missing image tag in screenshot
    at AppstreamScreenshot.checkTag (/home/travis/.nvm/versions/node/v10.15.3/lib/node_modules/@elementaryos/houston/dest/worker/task/appstream/screenshot.js:126:32)
    at Object.<anonymous> (/home/travis/.nvm/versions/node/v10.15.3/lib/node_modules/@elementaryos/houston/dest/worker/task/appstream/screenshot.js:110:76)
    at initialize.exports.each (/home/travis/.nvm/versions/node/v10.15.3/lib/node_modules/@elementaryos/houston/node_modules/cheerio/lib/api/traversing.js:300:24)
    at AppstreamScreenshot.<anonymous> (/home/travis/.nvm/versions/node/v10.15.3/lib/node_modules/@elementaryos/houston/dest/worker/task/appstream/screenshot.js:110:37)
    at step (/home/travis/.nvm/versions/node/v10.15.3/lib/node_modules/@elementaryos/houston/dest/worker/task/appstream/screenshot.js:49:23)
    at Object.next (/home/travis/.nvm/versions/node/v10.15.3/lib/node_modules/@elementaryos/houston/dest/worker/task/appstream/screenshot.js:30:53)
    at fulfilled (/home/travis/.nvm/versions/node/v10.15.3/lib/node_modules/@elementaryos/houston/dest/worker/task/appstream/screenshot.js:21:58)
```

Through a couple of attempts to fix it, the build got to pass again after removing those screenshots ([this commit](https://github.com/ryonakano/reco/commit/73cf21babea6df1ccbf5334cb1c162aa5ef2019b)).

## Expected Behavior

Houston could recognize localized screenshots and the build does not fails if developers add them.

**Comment by tintou** [comment]
https://github.com/elementary/houston/blob/58c871afbe9718d3bf59b70697dcf7f8e91c1558/src/worker/task/appstream/screenshot.ts#L63 this should probably be `if (image.length < 1) {`

**Comment by btkostner** [comment]
Fixed in 75ec27a3d39af2bd5ed23045b51f8e4e1d10b2dd. Thank you @tintou!

Should be deployed to `npm` (for travis ci testing) in less then 30 minutes, and on the production server within 12 hours.
